src/main/java/org/embulk/input/google_ads/GoogleAdsReporter.java

```diff
@@ -282,14 +286,65 @@ public List<String> buildWhereClauseConditions(PluginTask task, Map<String, Stri
     }
 
     public void connect()
+    {
+        this.client = buildClient(task.getLoginCustomerId().orElseGet(() -> getLoginCustomerId(task.getCustomerId())));
+    }
+
+    private Long getLoginCustomerId(String customerId)
+    {
+        try (CustomerServiceClient client = buildClient(null).getLatestVersion().createCustomerServiceClient()) {
+            return client.listAccessibleCustomers(ListAccessibleCustomersRequest.newBuilder().build())
+                    .getResourceNamesList()
+                    .stream()
+                    .map(CustomerName::parse)
+                    .map(CustomerName::getCustomerId)
+                    .map(this::getLoginCustomerClients)
+                    .flatMap(Collection::stream)
+                    .filter(loginCustomerClient -> customerId.equals(loginCustomerClient.customerClientId))
+                    .findFirst()
+                    .orElseThrow(() -> new RuntimeException("login customer not found [customer id: " + customerId + "]"))
```

Comment:
Reference:
https://support.google.com/google-ads/answer/7456530

```
A manager account can't be directly managed by more than 1 other manager account.
An individual Google Ads account can't be linked to more than one manager account in the same MCC hierarchy.
```

Due to the above limitations, only one login customer to be found.